### PR TITLE
Fix tab view wheel scroll

### PR DESCRIPTION
**Pull request by WinXaito**
## Pre-launch Checklist

- [x] I have updated `CHANGELOG.md` with my changes  
- [x] I have run "dart format ." on the project  
- [x] I have added/updated relevant documentation

### CODE REVIEW

**Comment by bdlukaa** [comment]
The `wheelScroll` property can then be removed, right?

**Comment by WinXaito** [comment]
> The `wheelScroll` property can then be removed, right?

Note necessary, if a user want to keep the vertical scroll over all page he can. 

But I think we must put the default value to true. 

What do you think about that ? And if you really want to remove it, I think it’s preferable to set it to true by default and mark as deprecated to avoid a breaking change.

**Comment by bdlukaa** [comment]
> And if you really want to remove it, I think it’s preferable to set it to true by default and mark as deprecated to avoid a breaking change and remove it in v5.0.0

Sounds fine to me! I checked on native and it stops the vertical scrolling, so I think we should do the same.

File: lib/src/controls/navigation/tab_view.dart [hunk 1]
```diff
@@ -74,6 +73,8 @@ class TabView extends StatefulWidget {
     this.header,
     this.footer,
     this.closeDelayDuration = const Duration(milliseconds: 400),
+    @Deprecated('this value is no longer used and will be removed in the next major release.')
```
Comment (bdlukaa) [inline lib/src/controls/navigation/tab_view.dart:76]:
```suggestion
    @Deprecated('This property is no longer used and will be removed in the next major release.')
```

**Review by bdlukaa — CHANGES_REQUESTED** [review]
Tests are failing because we make use of the property on the example app

**Comment by WinXaito** [comment]
👍 that must be good now

**Review by bdlukaa — APPROVED** [review]

**Pull request merged**